registration_form_with_db/templates/register.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Регистрация пользователя</title>
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/bootstrap.min.css') }}">
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
+</head>
+<body class="container mt-5">
+    <h2>Форма регистрации</h2>
+    {% with messages = get_flashed_messages() %}
+        {% if messages %}
+            <div class="alert alert-{{ messages[0].category }}">
+                {{ messages[0] }}
+            </div>
+        {% endif %}
+    {% endwith %}
+    <form method="post">
+        {{ form.csrf_token }}
+        <div class="form-group">
+            {{ form.first_name.label(class="form-label") }}
+            {{ form.first_name(class="form-control") }}
+            {% if form.first_name.errors %}
+                <small class="form-text text-danger">{{ form.first_name.errors[0] }}</small>
+            {% endif %}
+        </div>
+        <div class="form-group">
+            {{ form.last_name.label(class="form-label") }}
+            {{ form.last_name(class="form-control") }}
+            {% if form.last_name.errors %}
+                <small class="form-text text-danger">{{ form.last_name.errors[0] }}</small>
+            {% endif %}
+        </div>
+        <div class="form-group">
+            {{ form.email.label(class="form-label") }}
+            {{ form.email(class="form-control") }}
+            {% if form.email.errors %}
+                <small class="form-text text-danger">{{ form.email.errors[0] }}</small>
+            {% endif %}
+        </div>
+        <div class="form-group">
+            {{ form.password.label(class="form-label") }}
+            {{ form.password(class="form-control") }}
+            {% if form.password.errors %}
+                <small class="form-text text-danger">{{ form.password.errors[0] }}</small>
+            {% endif %}
+        </div>
+        <br>
+        <button type="submit" class="btn btn-primary">Регистрация</button>
```

Comment:
Не очень то оптимизированно. На лицо много повторений. Следуем принципу `DRY`
```suggestion
        {{ form.csrf_token }}
        {% for field in form if field.name not in ['csrf_token', 'submit'] %}
        <div class="form-group">
            {{ field.label(class="form-label") }}
            {{ field(class="form-control") }}
            {% for error in field.errors %}
                <small class="form-text text-danger">{{ error }}</small>
            {% endfor %}
        </div>
        {% endfor %}
        <br>
        {{ form.submit(class="btn btn-primary") }}
```